src/Pos/Txp/Toil/Types.hs

```diff
@@ -61,13 +61,13 @@ utxoF = later formatUtxo
 
 data BalancesView = BalancesView
     { _bvStakes :: !(HashMap StakeholderId Coin)
-    , _bvTotal  :: !Coin
+    , _bvTotal  :: !(Maybe Coin)
```

Comment:
I don't understand: why did you do it?
where will we use this `Maybe`?
what is goal of `Maybe` here, why not just 0 coins?